pacemaker.spec.in

```diff
@@ -169,7 +170,7 @@ It will run scripts at initialization, when machines go up or down,
 when related resources fail and can be configured to periodically check
 resource health.
 
-Available rpmbuild rebuild options:
+Available rpmbuild rebuild options (not necessarily applicable/with effect):
```

Comment:
Not true, as I tried to suggest with the previous iteration of this very change.
`--with cman` (or `--define` alternative) won't do anything on RHEL7.  This is the message of the comment, to avoid false assumptions about the magic miracles.

Honestly I am of the opinion that such an internal packaging detail should not be presented in the user-consumable description at all (yet to see a similar case).